script/doc-only-change.js

```diff
@@ -0,0 +1,58 @@
+const args = require('minimist')(process.argv.slice(2))
+const octokit = require('@octokit/rest')()
+const path = require('path')
+
+const SOURCE_ROOT = path.normalize(path.dirname(__dirname))
+
+async function checkIfDocOnlyChange () {
+  if (args.prNumber || args.prBranch || args.prURL) {
+    try {
+      let pullRequestNumber = args.prNumber
+      if (!pullRequestNumber || isNaN(pullRequestNumber)) {
+        if (args.prBranch) {
+          // AppVeyor doesn't provide a PR number for branch builds - figure it out from the branch
+          const prsForBranch = await octokit.pulls.list({
+            owner: 'electron',
+            repo: 'electron',
+            state: 'open',
+            head: `electron:${args.prBranch}`
+          })
+          if (prsForBranch.data.length === 1) {
+            pullRequestNumber = prsForBranch.data[0].number
+          } else {
+            // If there is more than one PR on a branch, just assume that this is more than a doc change
+            process.exit(0)
+          }
+        } else if (args.prURL) {
+          // CircleCI doesn't provide the PR number for branch builds, but it does provide the PR URL
+          const pullRequestParts = args.prURL.split('/')
```

Comment:
💡 can we perhaps directly extract the base branch from `CIRCLE_COMPARE_URL`?

Reply:
`CIRCLE_COMPARE_URL` isn't alway set, eg see: https://circleci.com/gh/electron/electron/348952